packages/mp/src/mp/core/data_models/connector/metadata.py

```diff
@@ -142,18 +142,21 @@ def _from_built(
         file_name: str,
         built: BuiltConnectorMetadata,
     ) -> ConnectorMetadata:
+        documentation_link = built.get("DocumentationLink")
+        if not documentation_link:
+            documentation_link = None
         return cls(
             file_name=file_name,
-            creator=built["Creator"],
+            creator=built.get("Creator", ""),
             description=built["Description"],
-            documentation_link=built["DocumentationLink"],
+            documentation_link=documentation_link,
```

Comment:
Why not just
```suggestion
            documentation_link=built.get("DocumentationLink"),
```

Let's verify if this is a must for connectors. If so we need to add a non fatal validation (separately) later on